src/app/shared/components/addscenario/addscenario.component.ts

```diff
@@ -81,6 +85,11 @@ export class AddScenarioModal implements OnInit, OnDestroy {
     }
 
     ngOnInit() {
+        this._nssService.currentItem.subscribe(item => this.itemParameters = item);
+        this._nssService.currentRegRegion.subscribe(regRegion => this.regRegion = regRegion);
+        this._nssService.currentStatisticGroup.subscribe(statisticGroup => this.statisticGroup = statisticGroup);
+        this._nssService.currentClone.subscribe(clone => this.clone = clone);
+
```

Comment:
As I mentioned in the mainview.component, if we combine everything into one object (say item), we'll only need one subscription here.